src/main/java/net/dv8tion/jda/api/requests/restaction/ThreadChannelAction.java

```diff
@@ -53,42 +50,6 @@ public interface ThreadChannelAction extends AuditableRestAction<ThreadChannel>
     @Override
     ThreadChannelAction deadline(long timestamp);
```

Comment:
No, because it clashes with `AuditableRestAction`.

Reply:
Since AuditableRestAction is pretty common, could we create a FluentAuditableRestAction via
`FluentAuditableRestAction<T, R> extends AuditableRestAction<T>, FluentRestAction<T, R>`
Outwardly people would just use AuditableRestAction